l10n/gitahead_ru.ts

```diff
@@ -0,0 +1,4999 @@
+        <translation>Имя темы</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="426"/>
+        <source>Restart?</source>
+        <translation>Перезапустить?</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="427"/>
+        <source>The application must be restarted for the theme change to take effect.</source>
+        <translation>Приложение должно быть перезапущено для того чтобы тема заработала.</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="429"/>
+        <source>Do you want to restart now?</source>
+        <translation>Хотите перезапустить сейчас?</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="431"/>
+        <source>Restart</source>
+        <translation>Перезапуск</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="432"/>
+        <source>Later</source>
+        <translation>Позже</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="450"/>
+        <source>Show full repository path</source>
+        <translation>Показывать полный путь к репозиторию</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="456"/>
+        <source>Hide automatically</source>
+        <translation>Скрывать автоматически</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="463"/>
+        <source>Open submodules in tabs</source>
+        <translation>Открыть подмодули во вкладках</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="469"/>
+        <source>Open all repositories in tabs</source>
+        <translation>Открыть все репозитории во вкладках</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="521"/>
+        <source>Theme:</source>
+        <translation>Тема:</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="522"/>
+        <source>Title:</source>
+        <translation>Название:</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="523"/>
+        <source>Log:</source>
+        <translation>Журнал:</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="524"/>
+        <source>Tabs:</source>
+        <translation>Вкладки:</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="526"/>
+        <source>Prompt:</source>
+        <translation type="unfinished"></translation>
```

Comment:
?

Reply:
IDK how to translate thi in this context. So let it be untranslated for now.